main/guides/getting-started/start-a-project.md

```diff
@@ -1,139 +1,177 @@
+`cd demo`
+`yarn install`
+
+![033](./assets/033.png)
+
+
+# Starting the Network
+Now go ahead and start the network using the `yarn start` command.
+`yarn start:docker`
+
+![034](./assets/034.png)
+
+Once the network has started you can check the logs. Once you see messages showing blocks with a status of `commit` you can rest assured the network is running properly.
+`yarn docker:logs`
+
+![035](./assets/035.png)
+
+
+# Starting the Dapp Smart Contract
+Exit the log window and start the smart contract by running the `yarn start` command.
+`yarn start:contract`
+
+![036](./assets/036.png)
+
+
+# Installing Keplr Wallet
+Next, you'll install the Keplr wallet plug-in. Open up your browser and navigate to [https://www.keplr.app/download](https://www.keplr.app/download). Select the version appropriate to your browser. 
+
+![037](./assets/037.png)
+
+Once the plug-in has been installed, open Keplr and select the option to "Import an existing wallet". Then choose the option to "Use recovery phrase or private key".
+
+![038](./assets/038.png)
 
-Leave this running in its own terminal window and visit [http://localhost:3000](http://localhost:3000) in a web browser.
+![040](./assets/040.png)
 
-## Connect the Dapp to the Agoric Wallet
+To import your wallet, you'll need to copy your mnemonic phrase into Keplr. You can find this series of 24 words back on your terminal window. Copy from this window into your Keplr wallet, then hit the "Import" button. Note that your phrase will NOT be the same as the one shown in this guide, all phrases are unique!
 
-1. Once here, you will be asked to enable the Dapp in your Agoric wallet.
+![039](./assets/039.png)
 
-![Dapp card store ui - needs approval](./assets/must-enable-dapp.png)
+![041](./assets/041.png)
 
-1. Navigate back to [http://127.0.0.1:8000](http://127.0.0.1:8000) and accept the Dapp's request to connect to your wallet.
+Give your new wallet a name and a password.
 
-![agoric wallet - Dapp approval prompt](./assets/accept-dapp-connection.png)
+![042](./assets/042.png)
 
-2. Navigate back to [the Dapp](http://localhost:3000) and it should load the baseball cards to bid on
+Ensure the "Cosmos Hub" chain is selected, then click "Save".
 
-![Dapp card store ui - loading cards](./assets/card-store-ui.png)
+![043](./assets/043.png)
 
-## Use the Dapp to bid on and buy a baseball card
+Starting the Dapp
+To start the UI for the sample dapp, run the `yarn start:ui` command. Note the localhost link that appears on your terminal window. Open this link in your browser.
 
-1. In the Dapp, you should be able to click on a baseball card to `BID` on it in an action. Enter `Bid ammount` to submit an offer to buy the card.
+![044](./assets/044.png)
 
-   ![Bid on Card](./assets/bid-on-card.png)
+![045](./assets/045.png)
 
-1. In the wallet, `Approve` the `Proposed` offer to bid on a card.
+From the browser UI, click the "Connect Wallet" button to connect your Keplr wallet. You will be asked to approve this connection.
```

Comment:
Good; I think I didn't manage to document the "approve this connection" step.

Reply:
:)